Dockerfile

```diff
@@ -660,3 +666,6 @@ EOT
 # > SYSTEMD=true make shell
 FROM dev-base AS dev
 COPY --link . .
+
+FROM dev AS devcontainer
+COPY --link --from=gopls     /build/ /usr/local/bin/
```

Comment:
Missing a newline here.

But we should probably check as well if this one should go before the `dev` stage. ISTR the idea here was for the final stage to be `dev`, so that building the Dockerfile without specifying a target would build the `dev` stage as the default one.

Reply:
Ah right, good point. I'll swap them.

Reply:
That said; if `gopls` would be useful altogether in the `dev` container, we could also choose to add it to the `dev` stage.

Reply:
Yeah I considered it but I don't think anyone actually runs the editor and a language server inside the shell container so IMO it doesn't make sense to force it to depend on the `gopls`.